x/evm/client/cli/query.go

```diff
@@ -374,34 +374,51 @@ func getCmdBurnerInfo(queryRoute string) *cobra.Command {
 }
 
 // getCmdChains returns the query to get all EVM chains
-func getCmdChains(queryRoute string) *cobra.Command {
+func getCmdChains() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "chains",
-		Short: "Get EVM chains",
+		Short: "Return the registered EVM chains",
 		Args:  cobra.ExactArgs(0),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			clientCtx, err := client.GetClientQueryContext(cmd)
-			if err != nil {
-				return err
-			}
+	}
 
-			queryClient := types.NewQueryServiceClient(clientCtx)
+	status := cmd.Flags().String("status", "", "the chain status [all|activated|deactivated]")
```

Comment:
make the cases string constants and reuse for both the description and switch statement